src/app/globals.css

```
@@ -2,245 +2,256 @@
+  --border: #f4d3e0;
+  --background-rgb: 255 246 238;
+  --foreground-rgb: 74 52 44;
+  --primary-rgb: 255 158 199;
+  --primary-strong-rgb: 240 107 168;
+  --secondary-rgb: 185 167 242;
+  --accent-rgb: 126 221 180;
+  --card-rgb: 255 255 255;
+  --border-rgb: 244 211 224;
+  --soft-shadow: 0 10px 30px -8px rgba(240, 107, 168, 0.18), 0 4px 10px -4px rgba(74, 52, 44, 0.08);
+  --lift-shadow: 0 18px 40px -12px rgba(240, 107, 168, 0.28), 0 6px 14px -6px rgba(74, 52, 44, 0.10);
   color-scheme: light;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)